### PR TITLE
chore(i18n): init crowdin config with replit plugin

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: .github/workflows/crowdin-upload.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Crowdin Upload
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every day at 11:00 AM UTC
+    - cron: "0 11 * * *"
```
Comment (ShaunSHamilton) [inline .github/workflows/crowdin-upload.yml:6]: @nhcarrigan I am thinking this is a bit severe for this project. I would _maybe_ upload once per week. Thoughts?

File: .github/workflows/crowdin-download.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          uses: actions/checkout@v2
+          with:
+            token: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+
+        - name: Generate Crowdin Config
+          uses: freecodecamp/crowdin-action@main
+          env:
+            CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+            CROWDIN_API_URL: "https://freecodecamp.crowdin.com/api/v2/"
+            CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_RIR }}
```
Comment (ShaunSHamilton) [inline .github/workflows/crowdin-download.yml:24]: Now that I have changed this, where should it go?

File: .github/workflows/crowdin-upload.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v2
+
+      - name: Generate Crowdin Config
+        uses: freecodecamp/crowdin-action@main
+        env:
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+          CROWDIN_API_URL: "https://freecodecamp.crowdin.com/api/v2/"
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
```
Comment (nhcarrigan) [inline .github/workflows/crowdin-upload.yml:21]:
```suggestion
          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_RIR }}
```

To match the others?

File: .github/workflows/crowdin-upload.yml (hunk 1 above)
Comment (nhcarrigan) [reply .github/workflows/crowdin-upload.yml:6]: If you're not anticipating a lot of changes to the course, we could definitely scale back the CRON job - or even disable it and only run the upload manually.

Comment (ShaunSHamilton) [reply .github/workflows/crowdin-upload.yml:6]: I say we do it manually.

Comment (nhcarrigan) [inline .github/workflows/crowdin-upload.yml:6]:
```suggestion
  # schedule:
    # runs every day at 11:00 AM UTC
    # - cron: "0 11 * * *"
```
comment it out so we just have workflow_dispatch for manual runs

File: .github/workflows/crowdin-download.yml (hunk 2 above)
Comment (nhcarrigan) [reply .github/workflows/crowdin-download.yml]: I loaded both of these into the repo secrets

**Review by nhcarrigan — APPROVED** [review]
Okay this should all be good to go. Will merge and run the upload.

**Pull request merged**